app/models/manageiq/providers/redhat/infra_manager/refresh_parser.rb

```diff
@@ -658,4 +659,10 @@ def self.datacenter_inv_to_hashes(inv, cluster_uids, vm_uids, storage_uids, host
 
     result
   end
+
+  private_class_method
```

Comment:
Thank for the blog post, its a very nice idea.
It will take me a while to refactor a 600+ lines class and make sure for each method no one else uses it.

Are we going for the all or nothing approach?

The blog post itself talks about breaking the method into smaller ones as an option, saying it is not nice because of the class << self ugly syntax which is not actually required any more, (though I agree the instance approach feels more oop in general)
The syntax I proposed is not confusing or unclear. It is different from the pattern, but the pattern is wrong, private stuff should be marked as private in some way.

So I think making this change is one step in a better direction, not perfect just better.
But I will not insist, technically its just a matter of deleting this line.